pyrsistent/_helpers.py

```diff
@@ -3,8 +3,12 @@
 from pyrsistent._pset import PSet, pset
 from pyrsistent._pvector import PVector, pvector
 
+# get PVector and PMap types for freeze - PVector type is different if C
+# extension has loaded
+pmap_type = type(pmap())
+pvector_type = type(pvector())
 
-def freeze(o):
+def freeze(o, nonstrict=False):
```

Comment:
How about calling it `strict` instead with a default value of `True`? Would get rid of some of the double negation below.